skyportal/tests/api/test_shifts.py

```diff
@@ -34,7 +34,7 @@ def test_shift(public_group, super_admin_token, view_only_token, super_admin_use
 
     assert any(
         [
-            len([s for s in s['users'] if s['id'] == super_admin_user.id]) == 1
+            len([s for s in s['shift_users'] if s['id'] == super_admin_user.id]) == 1
             for s in data['data']
```

Comment:
using `s` as the looping object in both internal and external loop is a little dangerous. Maybe rename one of them?

Reply:
You're completely right, github copilot naming of variables makes us lazy 👀